### DISCUSSION TITLE
Please persist simBrief user name for flyPad

### DISCUSSION THREAD

**Discussion by JensKn** [flybywiresim/aircraft#4979] [Ideas]
While simBrief integration into flyPad works like a charm, it could be improved by 

- persisting the user name or 
- using the same account information which is already persisted for usage by the MCDU.

Both ways prevent the user from entering the simBrief user name before each flight.

**Comment by scoopscode** [comment]
I believe this will be added or is already added in FlyPad OS 2

**Reply by JensKn** [reply]
Glad to hear that this is already addessed. Thanks for your answer!
